src/Microsoft.Health.Fhir.SqlServer/Features/Search/SqlServerSearchService.cs

```diff
@@ -1028,46 +1018,36 @@ private async Task<SearchResult> SearchForReindexSurrogateIdsWithoutSearchParamH
             long endResourceSurrogateId = 0;
 
             SearchResult searchResult = null;
-            await _sqlRetryService.ExecuteSql(
-                async (cancellationToken, sqlException) =>
-                {
-                    using SqlConnection connection = await _sqlConnectionBuilder.GetSqlConnectionAsync(initialCatalog: null, cancellationToken: cancellationToken).ConfigureAwait(false);
-                    using SqlCommand sqlCommand = connection.CreateCommand();
-                    connection.RetryLogicProvider = null; // To remove this line _sqlConnectionBuilder in healthcare-shared-components must be modified.
-                    await connection.OpenAsync(cancellationToken);
 
-                    sqlCommand.CommandTimeout = Math.Max((int)_sqlServerDataStoreConfiguration.CommandTimeout.TotalSeconds, 180);
-
-                    sqlCommand.Parameters.AddWithValue("@p0", resourceTypeId);
-                    sqlCommand.CommandText = @"
+            using var sqlCommand = new SqlCommand();
+            sqlCommand.CommandTimeout = Math.Max((int)_sqlServerDataStoreConfiguration.CommandTimeout.TotalSeconds, 180);
+            sqlCommand.Parameters.AddWithValue("@p0", resourceTypeId);
+            sqlCommand.CommandText = @"
                         SELECT ISNULL(MIN(ResourceSurrogateId), 0), ISNULL(MAX(ResourceSurrogateId), 0), COUNT(ResourceSurrogateId)
                         FROM dbo.Resource
                         WHERE ResourceTypeId = @p0
                             AND IsHistory = 0
                             AND IsDeleted = 0";
```

Comment:
Please replace with single line
SELECT isnull(min(ResourceSurrogateId), 0), isnull(max(ResourceSurrogateId), 0), count(*) FROM dbo.Resource WHERE ResourceTypeId = @p0 AND IsHistory = 0 AND IsDeleted = 0

Reply:
I usually use lower case for system functions like isnull, max/min, count, etc.